examples/static/microphone-streaming-auto-stop.html

```diff
@@ -34,7 +34,6 @@
 
       var stream = WatsonSpeech.SpeechToText.recognizeMicrophone({
         token: token,
-//        continuous: false, // no longer supported
```

Comment:
We should either remove this entire example or else make it work as advertised (add some code to call `.stop()` on the stream at the first `final` result.)

Fixing the example is, of course, preferred. But deleting it is fine if you don't have the time. If you delete it, also delete the link in `index.html`.